cmake/functions.cmake

```diff
@@ -9,8 +9,7 @@ function(addtest test_name)
   add_executable(${test_name} ${ARGN})
   addtest_part(${test_name} ${ARGN})
   target_link_libraries(${test_name}
-      GTest::main
-      GMock::main
+      GTest::gmock_main
```

Comment:
Does gmock_main includes gtest_main or gtest::main?

Reply:
It is modern way is using in new version